fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/MfdFmsPerf.tsx

```diff
@@ -415,6 +425,10 @@ export class MfdFmsPerf extends FmsPage<MfdFmsPerfProps> {
 
   private missedEngineOutAccelAltIsPilotEntered = Subject.create<boolean>(false);
 
+  private unit = Subject.create<Unit<UnitFamily.Distance>>(
+    NXDataStore.get('CONFIG_USING_METRIC_UNIT') === '1' ? UnitType.METER : UnitType.FOOT,
+  );
```

Comment:
```suggestion
  private lengthUnit = Subject.create<Unit<UnitFamily.Distance>>(UnitType.METER);
```

Reply:
I did this inititially, and it means that it will always be Meter on first render, even if you have set LBS on Flypad, as the subscribe is onAfterRender() and no value updates to set the LengthUnit differently

Reply:
Its unclear to me if this will still be a potential thing or not